### ISSUE TITLE
A question about pretrained model

### ISSUE THREAD

**Issue by galaxyWisdom** [pengzhiliang/MAE-pytorch#35]
hi, I have a question that if I want to extract the feature in pretrained model,should I only to inference the encoder with  all mask=0?

**Comment by pengzhiliang** [comment]
It's a good question, and I will test it later.

**Comment by pengzhiliang** [comment]
I have test it. You can set the `--mask_ratio=0.` in the command.
But you will get an ERROR when model calculates the mse loss.
To avoid it, you can add a `evaluate_one_epoch` fuction in `engine_for_pretraining.py` to extract the feature of pretrained model. And maybe [def evaluate(data_loader, model, device)](https://github.com/pengzhiliang/MAE-pytorch/blob/3546179ac5c4f0b80918b79f68678a42c8719808/engine_for_finetuning.py?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L151) can help you.
